syntax = "proto3";
option cc_enable_arenas = true;

package NetProto;

enum ProtoId
{
	PID_Min = 0;
	PID_Ping = 1;
	PID_Pong = 2;
	
	PID_QueryFreeHero = 100;
	PID_RspFreeHero = 101;
	PID_SelectHeroReq = 102;
	PID_SelectHeroRsp = 103;
	PID_LoadSceneComplete = 104;
	PID_LeaveScene = 105;
	PID_PullAllSceneInfo = 106;
	PID_PullAllSceneInfoRsp = 107;
	
	PID_SceneObjectState = 1000;
	PID_MoveObjectState = 1010;
	PID_MoveObjectMutableState = 1011;
	PID_MoveToPos = 1020;
	PID_StopMove = 1030;
	
	PID_BattleOperaReq = 1100;
	PID_ViewSnapshot = 1110;
	PID_ViewAllGrids = 1111;
	
	PID_Max = 5120;
}